src/debugger/SymbolManager.hh

```diff
@@ -72,14 +78,15 @@ public:
 	//   existing file is not replaced and this method returns 'false'.
 	//   Otherwise it return 'true'.
 	enum class LoadEmpty { ALLOWED, NOT_ALLOWED };
-	bool reloadFile(const std::string& filename, LoadEmpty loadEmpty, SymbolFile::Type type);
+	bool reloadFile(const std::string& filename, LoadEmpty loadEmpty, SymbolFile::Type type, std::optional<uint16_t> slot = {}, std::optional<uint16_t> subslot = {});
 
 	void removeFile(std::string_view filename);
 	void removeAllFiles();
 
-	[[nodiscard]] const auto& getFiles() const { return files; }
+	[[nodiscard]] auto& getFiles() { return files; }
+	[[nodiscard]] std::optional<int> findFile(std::string filename);
 	[[nodiscard]] std::span<Symbol const * const> lookupValue(uint16_t value);
-	[[nodiscard]] std::optional<uint16_t> parseSymbolOrValue(std::string_view s) const;
+	[[nodiscard]] std::optional<uint32_t> parseSymbolOrValue(std::string_view s) const;
```

Comment:
These are required because of the 24bit address (segment + address).
![image](https://github.com/openMSX/openMSX/assets/7815819/f1ece55e-690f-4ee9-830e-5aa857a116ae)